### PR TITLE
feat: terrain db and ND rendering

### PR DESCRIPTION
**Pull request by svengcz**
- introduce interpreter of FBW-terrain db files
- introduce Honeywell-compatible ND map renderer

### CODE REVIEW

File: apps/server/src/terrain/dto/ndterraindata.dto.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsNumber } from 'class-validator';
+
+/* eslint-disable camelcase */
+export class NDTerrainDataDto {
```
Comment (Lucky38i) [inline apps/server/src/terrain/dto/ndterraindata.dto.ts:5]: Possible to have new dto name that has more than 2 characters per word?

File: apps/server/src/main.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ const dirs = [
     'resources/coroutes',
     'resources/pdfs',
     'resources/images',
+    'resources/terrain',
```
Comment (Lucky38i) [inline apps/server/src/main.ts:23]: This will need to be its own folder for fragment to handle

File: apps/server/src/terrain/dto/ndview.dto.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsBoolean, IsNumber } from 'class-validator';
+
+/* eslint-disable camelcase */
+export class NDViewDto {
```
Comment (Lucky38i) [inline apps/server/src/terrain/dto/ndview.dto.ts:5]: Same here

**Review by Lucky38i — CHANGES_REQUESTED** [review]
Minor things

File: apps/server/src/main.ts (hunk 2 above)
Comment (svengcz) [reply apps/server/src/main.ts:23]: Outside of resources?

Comment (Lucky38i) [reply apps/server/src/main.ts:23]: Actually just get rid of it. That is for creating folders.

Comment (svengcz) [reply apps/server/src/main.ts]: done

**Review by Lucky38i — APPROVED** [review]

**Pull request merged**